codespell_lib/data/dictionary.txt

```diff
@@ -8933,8 +8945,13 @@ halp->help
 halpoints->halfpoints
 hammmer->hammer
 hampster->hamster
+handel->handle
 handeld->handled, handheld,
 handeled->handled, handheld,
+handeler->handler
+handeles->handles
+handeling->handling
+handels->handles
```

Comment:
His music (Handel's)?

Reply:
This is a usecase for using the `--ignore` flag.